admin/console/other-useful-commands.rst

```diff
@@ -13,29 +13,47 @@ This will also show errors that might appear within that process.
 
    >> Channel.fetch
 
-Reprocess unprocessable mails
------------------------------
+Reprocess Unprocessable Emails
+------------------------------
 
-When Zammad encounters a mail it cannot parse (e.g. due to a parser bug or a
-malformed message), it will store the mail in
-``var/spool/unprocessable_mail/<ID>.eml``,  give up on attempting to parse the
-mail, and will warn on the monitoring page that there are unprocessed mails.
-
-To force Zammad to reattempt to parse those mails, run the following command:
-
-.. code-block:: ruby
-
-   >> Channel::EmailParser.process_unprocessable_mails
+When Zammad fetches a mail it cannot parse (e.g. due to a parser bug or a
+malformed message), it will store the mail in the database and warn in
+the monitoring section about it.
 
 In case of a malformed message (e.g. an invalid email address in one of the
 header fields), you may need to manually edit the mail before Zammad can process
-it.
-
-If Zammad fails to process the message, it will remain in the
-``var/spool/unprocessable_mail`` folder; otherwise it will be removed after it
-has been parsed successfully.
-
-Add translation
+it. To do so, follow the steps below:
+
+Export all Failed Emails to a Local Folder
+   Execute ``rake zammad:email_parser:failed_email:export_all``. You can find
+   the location of the exported email in the output of your console.
+   Every time you perform an export of unprocessable emails, it creates one
+   folder with all unprocessable emails at the time of execution in it.
+
+Edit the Email
+   The email has been exported in the step above. Now you can have a look at it
+   and try to repair it. Make sure to leave the file name untouched, as the
+   import will otherwise fail.
+
+Import Locally Modified Email Back to Database
+   After editing the email, run
+   ``rake zammad:email_parser:failed_email:import /path/to/your/email.eml``
+   to apply your changes from the file to the database.
```

Comment:
We should mention that you can also pass the entire folder as well, and all .eml files in it will be imported.